streampark-flink/streampark-flink-kubernetes/src/main/scala/org/apache/streampark/flink/kubernetes/ingress/IngressController.scala

```diff
@@ -30,8 +30,7 @@ object IngressController extends Logger {
   private lazy val ingressStrategy: IngressStrategy = {
```

Comment:
defined before line 30:
`private[this] val VERSION_REGEXP = "^v(\\d+\\.\\d+)".r`

Reply:
> 

completed